Sources/Generics/GenericTableViewController.swift

```diff
@@ -71,6 +71,7 @@ UISearchResultsUpdating, UISearchControllerDelegate {
         super.viewWillAppear(animated)
         tableView.reloadData()
         scrollToSelectedRow()
+        searchController.searchBar.setLeftAligment()
```

Comment:
move this logic to func setupUISearchController()

Reply:
fixed